### PR TITLE
chore: simplify release rpm creation

### PR DESCRIPTION
**Pull request by SpyTec**
Depends on #1024 

In action:
https://github.com/SpyTec/convert2rhel/releases/tag/v04.19.15

Jira Issues: [RHELC-](https://issues.redhat.com/browse/RHELC-)

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [ ] `[RHELC-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [ ] PR title explains the change from the user's point of view
- [ ] Code and tests are documented properly
- [ ] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,21 +35,13 @@ jobs:
         ### end manpage generation
 
       - name: Build RPM package for EL7
-        id: rpm_build_el7
-        uses: oamg/rpmbuild@el7
-        with:
-          spec_path: "packaging/convert2rhel.spec"
+        run: |
+          make rpm7
```
Comment (r0x0d) [inline .github/workflows/release.yml:39]: Why not turning this into a matrix and simplifying even further?

Comment (SpyTec) [reply .github/workflows/release.yml:39]: Given the automate manpage generation that @Andrew-ang9 is going to be working on I figured it's best to leave it for that. But yeah I'll just simplify it now (and also enable el9 builds) then Andrew can focus on the manpage generation commit in pre-commit

Comment (SpyTec) [reply .github/workflows/release.yml]: Done

**Review by r0x0d — APPROVED** [review]
looks lovely!

**Pull request merged**